### PR TITLE
chore: remove proto library tests from CI

### PR DESCRIPTION
**Pull request by aignas**
Cleanup after proto libraries moved out of rules_python.

### CODE REVIEW

**Review by groodt — APPROVED** [review]
LGTM

**Comment by rickeylev** [comment]
It looks like we still have a full implementation in rules_python, though? I figured python/proto.bzl would load from the protobuf repo at some time to transition

**Comment by aignas** [comment]
Ah, I forgot we reverted the py_proto_library removal. Then let's wait until that change gets merged again.

**Pull request closed without merging**
